kernel/arch/dreamcast/include/dc/pvr.h

```diff
@@ -2131,6 +2133,52 @@ void pvr_dma_init(void);
 /** \brief  Shut down PVR DMA. */
 void pvr_dma_shutdown(void);
 
+/** \brief   Copy a block of memory to VRAM
+    \ingroup store_queues
+
+    This function is similar to sq_cpy(), but it has been
+    optimized for writing to a destination residing within VRAM.
+
+    \warning
+    This function cannot be used at the same time as a PVR DMA transfer.
+
+    The dest pointer must be at least 32-byte aligned and reside 
+    in video memory, the src pointer must be at least 8-byte aligned, 
+    and n must be a multiple of 32.
+
+    \param  dest            The address to copy to (32-byte aligned).
+    \param  src             The address to copy from (32-bit (8-byte) aligned).
+    \param  n               The number of bytes to copy (multiple of 32).
+    \param  type            The type of SQ/DMA transfer to do (see list of modes).
+    \return                 The original value of dest.
+
+    \sa sq_cpy()
```

Comment:
Same here. Im curious if these will generate Doxygen warnings not that they are in a different file (sq.c)?

Great job!!!! Functions are more flexible now.